application/views.py

```diff
@@ -0,0 +1,105 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+from .forms import HighSchoolApplicationForm
+from .models import HighSchoolApplication
+from django.utils import timezone
+from django.urls import reverse
+
+
+def new_application(request):
+    if request.method == "POST":
+        # user_id will be replaced by sessions
+        user_id = 1
+        form = HighSchoolApplicationForm(request.POST)
+        if form.is_valid():
+            f = form.save(commit=False)
+            f.user_id = user_id
+            f.application_number = (
+                str(user_id) + str(f.school.school_name) + str(f.program)
+            )
+            f.submitted_date = timezone.now()
+            try:
+                if "Submit" in request.POST.get("submit"):
+                    f.is_draft = False
+                else:
+                    f.is_draft = True
+            except:  # noqa: E722
+                pass
+            f.save()
+            return HttpResponseRedirect(reverse("application:index"))
+    else:
+        form = HighSchoolApplicationForm()
+    context = {"form": form}
+    return render(request, "application/application-form.html", context)
+
+
+def save_existing_application(request, application_id):
+    if request.method == "POST":
+        form = HighSchoolApplicationForm(request.POST)
+        if form.is_valid():
+            user_id = 1
```

Comment:
Please add a TODO for not hard coding this and use the sessions object to get the value of user_id.